packages/openassistants/openassistants/eval/eval_interaction.py

```diff
@@ -0,0 +1,244 @@
+import abc
```

Comment:
Is there a reason we switched from `openassistants.eval.interaction` to `openassistants.eval.eval_interaction`?  I thought the previous filename was reasonable, and don't know what this refactor does other than make the diff harder to reason over.

Reply:
fixed. ugh pre commit hooks / static analysis are not catching these.